### ISSUE TITLE
Spermiation failure

### ISSUE THREAD

**Issue by MagoWyr** [obophenotype/human-phenotype-ontology#9825]
**Preferred term label:**
Spermiation failure

**Synonyms**

**Definition (free text, please give PubMed ID)**
Very seldomly “Spermiation failure”, a condition characterised by interrupted release of the sperm, is the predominant cause of azoospermia. Spermiation failure is indicated by apparently normal spermiogenesis, but the presence of retained Sd2 spermatids in stages III-V of human spermatogenesis, whereas maturation arrest is indicated by a lack of late Sd1 and Sd2 spermatids in the epithelium. It is important to note that multiple spermatogenic abnormalities often occur simultaneously in infertile men, and the helical nature of human spermatogenesis can make it difficult to readily discriminate multiple abnormalities of spermatid development and release.

**Parent term (use hpo.jax.org/app)**
HP:0011961

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA: 399771

**Your nano-attribution (ORCID)**
0000-0002-7651-2403

**Comment by pnrobinson** [comment]
@MagoWyr 
added as HP:0034812
